spec/mailers/user_mailer_spec.rb

```diff
@@ -432,4 +432,76 @@
       end
     end
   end
+
+  describe "invited_to_collection_notification" do
+    let(:collection) { create(:collection) }
+    let(:otheruser) { create(:user) }
+    let(:work) { create(:work, authors: [otheruser.default_pseud]) }
+
+    let(:email) { UserMailer.invited_to_collection_notification(otheruser.id, work.id, collection.id).deliver }
+
+    # Test the headers
+    it_behaves_like "an email with a valid sender"
+
+    it "has the correct subject line" do
+      text = "[#{ArchiveConfig.APP_SHORT_NAME}][#{collection.title}] Request to include work in a collection"
+      expect(email.subject).to eq(text)
+    end
+
+    # Test both body contents
+    it_behaves_like "a multipart email"
+    
+    it_behaves_like "a translated email"
+
+    describe "HTML version" do
+      it "has the correct content" do
+        expect(email).to have_html_part_content("would like to include your work")
+      end
+    end
+
+    describe "text version" do
+      it "has the correct content" do
+        expect(email).to have_text_part_content("would like to include your work")
+      end
+    end
+  end
+
+  describe "added_to_collection_notification" do
+    let(:collection) { create(:collection) }
+    let(:otheruser) { create(:user) }
+    let(:work) { create(:work, authors: [otheruser.default_pseud]) }
+
+    let(:email) { UserMailer.added_to_collection_notification(otheruser.id, work.id, collection.id).deliver }
```

Comment:
Could you update this to `subject(:email)` as well?

Reply:
Done